provisioning/templates-api/kubernetes/templates/api/deployment.yaml

```diff
@@ -44,15 +44,19 @@ spec:
                 configMapKeyRef:
                   name: templates-api
                   key: storageApiHost
-            - name: ETCD_ENDPOINT
+            - name: TEMPLATES_API_ETCD_ENABLED
+              value: true
```

Comment:
`TEMPLATES_API_ETCD_ENABLED` true je default hodnota, ale dal som to sem radsej explicitne.